### DISCUSSION TITLE
Data Visualizations - Colors

### DISCUSSION THREAD

**Discussion by benjamincoughlin** [twilio-labs/paste#2496] [Office hours] [answered]
I am working on updating the data visualizations for Video Insights within the Console 
& one of the requests is to modify the colors for better accessibility. 

The current data viz Paste Color palette appears to only go from 40-90 for a value range making everything quite dark & heavy across the various screens. There is also some differences between the Paste Website colors and the Figma Library Colors.

Any guidance would be much appreciated & I'll also be signing up for office hours.

I will be moving away from the solid fills as a start to make things better, similar to either one of these Storybook examples:
https://5e53448165911c0022e68c74-lbvvmxmazl.chromatic.com/?path=/story/libraries-data-visualization--basic-area-chart
https://5e53448165911c0022e68c74-lbvvmxmazl.chromatic.com/?path=/story/libraries-data-visualization--time-series

Current Paste Colors:
<img width="1017" alt="Screen Shot 2022-06-27 at 9 17 52 AM" src="https://user-images.githubusercontent.com/102981799/175989009-afe19b97-eab3-4ecd-bd33-07e3e44a1d66.png">
![Screen Shot 2022-06-27 at 9 17 21 AM](https://user-images.githubusercontent.com/102981799/175989026-714aa973-a263-47c2-bca8-839294878ce9.png)

Video Insights with Figma Library color palette example:
![Video Insights - Current Palette Example](https://user-images.githubusercontent.com/102981799/175989623-ae861d41-2b99-41f0-93f1-30b3d0214470.png)

@kkenna3

**Comment by TheSisb** [comment, marked as answer]
Hi @benjamincoughlin,

I think I spot two different questions, let me know if I missed something:

> The current data viz Paste Color palette appears to only go from 40-90 for a value range making everything quite dark & heavy across the various screens. 

Unfortunately, none of our color aliases below 40 on the scale are accessible for graphical elements. When you say "one of the requests is to modify the colors for better accessibility", these colors will do the job!


> There is also some differences between the Paste Website colors and the Figma Library Colors.

So the reason they look different on the website is that the data visualization token colors change based on the theme. Your website appears to be in the `dark` theme, while Figma is showing the tokens for the `default` theme.

Hope this helps!

**Comment by gloriliale** [comment]
Hi there! Wanted to leave the recording of Office Hours where we discussed this here for reference: [Office Hours recording](https://www.loom.com/share/cfaf9c692ccb4fecaeb62b2625aab9f9). The password is posted in the #help-design-system channel. Thanks again for coming to office hours!
